calicoctl/calico_ctl/node.py

```diff
@@ -574,6 +575,9 @@ def node_stop(force):
     Stop the Calico node.  This stops the containers (calico/node and
     calico/node-libnetwork) that are started by calicoctl node.
     """
+    # The command has to be run as root to stop the calico-node service
```

Comment:
DO we need it on the stop?  The reason we need it on the start is because we need to load kernel modules.